src/components/TabbedBox.tsx

```diff
@@ -0,0 +1,96 @@
+import map = require('lodash/map');
+import * as React from 'react';
+import styled from 'styled-components';
+import Button from './Button';
+import { Box, Flex } from './Grid';
+import Txt from './Txt';
+
+interface TabbedBoxProps {
+	tabs: string[];
+	help?: JSX.Element;
+	children: JSX.Element[];
+}
+
+interface TabbedBoxState {
+	show: number;
+}
+
+const FlexTab = styled(Flex)`
+	border-style: hidden;
+	border-bottom: black;
```

Comment:
Can you please change the border color to match the border color we use on textareas and inputs?

Reply:
Done